### DISCUSSION TITLE
Google Drive Backups

### DISCUSSION THREAD

**Discussion by picocodes** [coollabsio/coolify#2724] [Feature Requests]
You currently support auto-backups to S3. It would be nice if the same was also possible for Google Drive.

**Comment by IRediTOTO** [comment]
Google drive is very popular and accessible. I have 2TB of Google drive for 10 years, I hope you will do this function.

**Comment by Ali7med** [comment]
it very helpful function

**Comment by k1rana** [comment]
Currently, I use n8n running on coolify to backup my database dump to Google Drive. S3 storage is running on coolify too. It might take a lot of resources, but this is one of the ways you might try.

Here is my workflow:
![Screenshot 2025-04-10 at 10 00 19](https://github.com/user-attachments/assets/b13863a6-f45c-4ddb-a96a-7662cc78dc25)

**Reply by joe-sal** [reply]
hey @k1rana , can you give us the json, without your sensitive data for this workflow ?

**Reply by IRediTOTO** [reply]
> Currently, I use n8n running on coolify to backup my database dump to Google Drive. S3 storage is running on coolify too. It might take a lot of resources, but this is one of the ways you might try.
> 
> 
> 
> Here is my workflow:
> 
> ![Screenshot 2025-04-10 at 10 00 19](https://github.com/user-attachments/assets/b13863a6-f45c-4ddb-a96a-7662cc78dc25)
> 
> 

can you share the json?

**Reply by k1rana** [reply]
@jxsflow @IRediTOTO I actually put that on my gist six months ago.
https://gist.github.com/k1rana/b74bd89c0535fd6e7b3053b501897c16

**Comment by lucasew** [comment]
That could be implemented using rclone

S3 is handled by a utility from minio

**Comment by ataapa9018** [comment]
I have 2TB of Google drive :D I love this feature
